src/screens/DiscoveryScreenV2/index.js

```diff
@@ -24,6 +24,7 @@ import {fonts} from '../../utils/fonts';
 import FollowingAction from '../../context/actions/following';
 import {Header} from '../../components';
 import {COLORS} from '../../utils/theme';
+import KeyboardWrapper from '../../navigations/KeyboardWrapper';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `KeyboardWrapper` is a new addition and seems to be used correctly in the `DiscoveryContainer` component. This change appears to be part of an effort to handle keyboard interactions more effectively.
```diff
+ import KeyboardWrapper from '../../navigations/KeyboardWrapper';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->